pmu.cpp

```diff
@@ -70,13 +70,13 @@ void PMU::open(const perf_event_attr &perf_config)
 
 	if (_fd < 0)
 	{
-		throw std::runtime_error("perf_event_open failed");
+		printf("perf_event_open failed");
```

Comment:
It is fine to only report a warning in this case, but `printf` won't work for Android. We'll need to use the logging macros on Android.
I think we should also log the counter id (from `_perf_config.config`), for slightly easier debugging.

Reply:
Done